### DISCUSSION TITLE
System font is not my cup of tea. Solutions?

### DISCUSSION THREAD

**Discussion by wiwigak** [sumatrapdfreader/sumatrapdf#3075] [Q&A]
Hello folks.

I have lots of ePub books to read and I love how fast the file opens in a blink of an eye with SumatraPDF. Impressive!

If I'm not mistaken, without embedded font, the latest Sumatra renders text in predfined system font.
But the system font fails to meet my expectations as beauty standard, especially when it renders the text in my own language.
I've been searching for a solution for this with no success.

Previous Sumatra allows for fontname declaration but drawback is obvious; it cannot copy text.

Here's the questions.

1. Is it by any chance possible to set the default font of my liking at this moment? If so, please tell me how.
2. Are you planning to work on implementing the fontname back into settings in the future release?
3. PDF file is in general faster in loading than ePub file, Is Sumatra optimized for viewing PDF rather than ebook formats?

Thank you again and wish you all the best.

**Comment by GitHubRulesOK** [comment]
Previous versions say 2.x-3.2? should in **fixed mode** have allowed find/copy/print ebook text but unsure how well that behaved for each font, some inferior fonts show boxes for characters.

MuPDF PDF/ePub/Fb2 engine attempts to use defined / embedded fonts or their preferred fallback

SumatraPDF speed will be the best possible with respect to balance resources for given document (frequently it is tuned)
